pybeerxml/utils.py

```diff
@@ -22,3 +22,10 @@ def cast_to_bool(value: Any) -> bool:
         return value
 
     return False
+
+
+def gravity_to_plato(gravity):
```

Comment:
Do you mind adding type hints here. I assume it could be:
```suggestion
def gravity_to_plato(gravity: float) -> float:
```